core/analytics-enabled/src/main/kotlin/com/wire/android/feature/analytics/AnonymousAnalyticsManagerImpl.kt

```diff
@@ -169,6 +172,17 @@ object AnonymousAnalyticsManagerImpl : AnonymousAnalyticsManager {
         }
     }
 
+    /**
+     * Check if the coroutine context is initialized as a safety measure, since we are late initializing this object.
+     */
+    private fun isCoroutineContextNotInitialized(): Boolean {
+        if (!::coroutineScope.isInitialized) {
```

Comment:
It could be more interesting to init lazy the coroutine scope inside, but the tests are not easy to fix. so given this is kinda a hotfix, I prefer to go with this option.